panic: reflect: call of reflect.Value.CanInterface on zero Value

When using struct fields that are pointers rendering panics if the value of that field is nil.

Example code:
<pre>
package main

import (
	"os"

	"github.com/landoop/tableprinter"
)

type person struct {
	Firstname *string `header:"first name"`
	Lastname  *string `header:"last name"`
}

func main() {
	printer := tableprinter.New(os.Stdout)

	first := "Firstname"

	persons := []person{
		{Firstname: &first},
	}

	printer.Print(persons)
}
</pre>

Results in:
<pre>
panic: reflect: call of reflect.Value.CanInterface on zero Value
</pre>

--- a/row.go
+++ b/row.go
@@ -151,7 +151,7 @@ func MakeFilters(in reflect.Value, genericFilters ...interface{}) (f []RowFilter
 }
 
 func extractCells(pos int, header StructHeader, v reflect.Value, whenStructTagsOnly bool) (rightCells []int, cells []string) {
-	if v.CanInterface() {
+	if v.IsValid() && v.CanInterface() {
 		s := ""
 		vi := v.Interface()
 

